test/ui-test/common/testUtils.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-await-in-loop */
+import { Notification, VSBrowser, NotificationsCenter, NotificationType, Workbench, SideBarView } from 'vscode-extension-tester';
+
+/**
+ * @author Ondrej Dockal <odockal@redhat.com>
+ */
+export async function findNotification(text: string): Promise<Notification | undefined> {
+  await new Workbench().openNotificationsCenter();
+  const notifications = await new NotificationsCenter().getNotifications(NotificationType.Any);
+  for (const notification of notifications) {
```

Comment:
I would prefer if we can try to keep the code more functional. With that in mind, `array.map` is better then `for..of`. 
You could change the `for .. of` to the following and it would be functional. 

```
  notifications.map(
    async (notification): Promise<Notification> => {
      if (notification) {
        const message = await notification.getMessage();
        if (message.includes(text)) {
          return notification;
        }
      }
    },
  );
```

Reply:
That would make you compliant with the linter and you can remove the disable comments at the top.